# Object Detection annotation data
Header header

#object details
string[] name
string[] pose
int16[] truncated
int16[] difficult

#bounding box
float32[] xmin
float32[] ymin
float32[] xmax
float32[] ymax
int16 totalObjectsInFrame